homeassistant/components/media_player/cast.py

```diff
@@ -17,6 +17,7 @@
 from homeassistant.core import callback
 from homeassistant.helpers.dispatcher import (dispatcher_send,
                                               async_dispatcher_connect)
+from homeassistant.components.cast import DOMAIN
```

Comment:
We should not overwrite the media_player domain with the cast domain. We can import it like so:
```py
from homeassistant.components.cast import DOMAIN as CAST_DOMAIN
```

Reply:
We're not importing the media player domain?

Reply:
The cast media_player platform belongs to the media_player domain. Importing the cast domain under the same name is confusing and not correct.

Reply:
fixed